test/integration_test/main_test.go

```diff
@@ -63,6 +64,11 @@ func setup() error {
 
 	ci_utils.PxUpgradeHopsURLList = strings.Split(pxUpgradeHopsURLs, ",")
 
+	ci_utils.PxOperatorVersion, err = ci_utils.GetPXOperatorVersion()
+	if err != nil {
+		return fmt.Errorf("failed to discover installed portworx operator version: %v", err)
+	}
+
```

Comment:
nit: If we are setting the variable `PxOperatorVersion` using a function also in the same package, should we just call `ci_utils.SetPxOperatorVersion` or `ci_utils.InitPxOperatorVersion` and let it set the value inside the function itself?

Reply:
the operator right now is installed out of the automation, so Get may work better for now. in the future if we install operator in the test, probably will change it to set.